catalog/MDCCatalog/MDCCatalogTiles.m

```diff
@@ -4115,15 +4114,15 @@ void MDCCatalogDrawTextFieldTile(CGRect frame, NSObject<MDCColorScheme> *colorSc
   CGRect textLabelRect = CGRectMake(CGRectGetMinX(textFieldGroup) + floor(textFieldGroup.size.width * 0.00000 + 0.5), CGRectGetMinY(textFieldGroup) + floor(textFieldGroup.size.height * 0.00000 + 0.5), floor(textFieldGroup.size.width * 0.27433 + 0.09) - floor(textFieldGroup.size.width * 0.00000 + 0.5) + 0.41, floor(textFieldGroup.size.height * 0.70270 + 0.5) - floor(textFieldGroup.size.height * 0.00000 + 0.5));
   NSMutableParagraphStyle* textLabelStyle = [[NSMutableParagraphStyle alloc] init];
   textLabelStyle.alignment = NSTextAlignmentLeft;
-  NSDictionary* textLabelFontAttributes = @{NSFontAttributeName: [UIFont fontWithName: TextFieldFontName size: 16], NSForegroundColorAttributeName: textForeground, NSParagraphStyleAttributeName: textLabelStyle};
+  NSDictionary* textLabelFontAttributes = @{NSFontAttributeName: TextFieldFontName, NSForegroundColorAttributeName: textForeground, NSParagraphStyleAttributeName: textLabelStyle};
```

Comment:
I believe that [`NSFontAttributeName`'s value should remain a UIFont object](https://developer.apple.com/documentation/foundation/nsattributedstringkey/1528839-font). Also it would be awesome (but not required) if you could reformat this line to get it under 100 columns. 😺